datafusion/sqllogictest/test_files/array.slt

```diff
@@ -4408,12 +4422,10 @@ select array_union(arrow_cast([], 'LargeList(Int64)'), arrow_cast([], 'LargeList
 query ?
 select array_union([[null]], []);
 ----
-[[NULL]]
+[[]]
 
-query ?
+query error DataFusion error: Error during planning: Failed to unify argument types of array_union:
```

Comment:
Is this a regression? It looks like the query used to run but now doesn't.

Reply:
It seems so @joroKr21 can we keep null?

Reply:
The problem is that the argument types don't make sense. How are we supposed to union `LargeList(List(Int64))` and `LargeList(Int64)`?